great_expectations/core/expectation_configuration.py

```diff
@@ -1378,9 +1383,22 @@ class ExpectationConfigurationSchema(Schema):
     ge_cloud_id = fields.UUID(required=False, allow_none=True)
     expectation_context = fields.Nested(lambda: ExpectationContextSchema)
 
+    @post_dump
+    def remove_null_values(self, data: dict, **kwargs):
+        # Will remove `ge_cloud_id` if not used
+        to_delete = []
+        for key, value in data.items():
+            if value is None:
+                to_delete.append(key)
+
+        for key in to_delete:
+            del data[key]
+
+        return data
+
```

Comment:
We could move this up to the parent `Schema` if we always want to omit `ge_cloud_id`. Thoughts on that?

Also happy to simply pull out the `ge_cloud_id` like this:
```python
    @post_dump
    def remove_null_values(self, data: dict, **kwargs):
        # Will remove `ge_cloud_id` if not used
        if "ge_cloud_id" in data and data["ge_cloud_id"] is None:
            del data["ge_cloud_id"]
        return data
```